airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -2509,8 +2511,8 @@ def execute(self, context: Context):
 class BigQueryInsertJobOperator(GoogleCloudBaseOperator):
     """
     Executes a BigQuery job. Waits for the job to complete and returns job id.
-    This operator work in the following way:
 
+    This operator work in the following way:
     - it calculates a unique hash of the job using job's configuration or uuid if ``force_rerun`` is True
```

Comment:
I think this would break list rendering. An empty line is needed after the `:` as well.